UnityProject/Assets/Scripts/Systems/Chemistry/ReagentMix.cs

```diff
@@ -295,6 +295,11 @@ public void Add(ReagentMix b)
 
 		public void Add(Reagent reagent, float amount)
 		{
+			if (amount == 0f)
+			{
+				return;
```

Comment:
```cs
if (Mathf.Approximately(amount, 0f)) return;
```

Reply:
is exactly 0 value Getting past in,
is normal covers tiny numbers